src/routes/basin_subzones.js

```diff
@@ -446,6 +446,38 @@ module.exports = (errorHandler, basinSubzoneService) => {
       })
   )));
 
+  /**
+   * @apiGroup s_ecoChange
+   * @api {get} /basinSubzones/:subzone_id/ecoChange/layers/lp/period/:period/categories/
+   * LPCategoriesLayerInSubzone
+   * @apiName LPCategoriesLayerInSubzone
+   * @apiVersion 0.1.0
+   * @apiDescription
+   * Get the the forest loss and persistence layer for a given period, divided by categories
+   * inside the basin subzone
+   *
+   * Data for 2000-2005, 2006-2010, 2011-2015, 2016-2019 periods
```

Comment:
Would be better to put these as possible values for the `period` param?

Reply:
Done